src/components/Provider/Analytics/ProviderStats/StatsFilter/StatsFilter.js

```diff
@@ -23,11 +24,20 @@ import { useIntl } from "react-intl";
  * @returns {JSX.Element} JSX Component.
  */
 const StatsFilter = ({ list, onChange }) => {
+  const baseExchanges = useExchangeList();
   const [exchange, setExchange] = useState("ALL");
   const [quote, setQuote] = useState("ALL");
   const intl = useIntl();
   const exchanges = useExchangesOptions(true);
-  const quoteAssets = useQuoteAssets();
+  const selectedBase =
+    baseExchanges &&
+    baseExchanges.find(
+      (item) => item.name.toLowerCase() === (exchange === "ALL" ? "binance" : exchange),
+    );
+  const quoteAssets = useBaseExchangeQuotes({
```

Comment:
useBaseExchangeQuotes is a bit confusing to me, because of the name I first thought it was also returning bases. Would useExchangeQuotes works?

Reply:
ahh yes, would userParentExchangeQuotes work?

Reply:
Why referring parent? The hook is to return the quotes of a passed exchange right?

Reply:
yeah, you are right, done.